poly-commitment/src/commitment.rs

```diff
@@ -39,7 +41,7 @@ use super::evaluation_proof::*;
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
 #[serde(bound = "C: CanonicalDeserialize + CanonicalSerialize")]
 pub struct PolyComm<C> {
-    #[serde_as(as = "Vec<o1_utils::serialization::SerdeAs>")]
+    #[serde_as(as = "Vec<o1_utils::serialization::SerdeAsUnchecked>")]
```

Comment:
Not sure if this changes breaks or no other code in develop/berkeley.

Reply:
Good catch, the change is not necessary. Will revert now.